### ISSUE TITLE
Feature request - notification on new chapters

### ISSUE THREAD

**Issue by waldemar-p** [sendyhalim/Yomu#10]
Greetings @sendyhalim 

First of all: You've made a very beautiful and great app. It's better that what I was looking for. And it would be at the pinnacle of manga reader apps if it would also notify me of new chapters. Now I'm not talking about all chapters that are being published in all mangas that exist, but only for the ones that are added in the persons list.

Thanks for your great work 👍

**Comment by sendyhalim** [comment]
@waldemar-p Yes, I'm planning to add some kind of notification badge (something like notification badge just like facebook notification icon) for new chapters. Right now I'm still focusing on the iOS version of Yomu, I'll get back to this after it's done 🍻, thanks for the kind words! 

In case you're interested to follow the development: https://github.com/sendyhalim/iyomu 🏃
